src/Rebing/GraphQL/Support/SelectFields.php

```diff
@@ -15,6 +15,7 @@
 use Illuminate\Database\Eloquent\Relations\MorphOne;
 use Illuminate\Database\Eloquent\Relations\BelongsTo;
 use Illuminate\Database\Eloquent\Relations\MorphMany;
+use Illuminate\Database\Eloquent\Relations\MorphOne;
```

Comment:
This cases travis to fail, the relation is already imported on line 16

Reply:
has been revised